auto_execution: true
b_root: [0.3952562663771974, 0.033508481949253585, 0.07600197996895874]
target_l_offset: [0.002, -0.01, 0.0]
target_r_offset: [-0.009, 0.008, 0.002]
target_id: 0
l_l_offset: [0, -0.008, 0.011]
l_r_offset: [-0.005, 0.01, 0.004]
r_root: [0.38045453044240995, 0.03312732487576066, 0.07048050567937017]
sites_availabilty:
- [0.0, 0.0]
- [0.0, 1.0]
- [0.0, 0.0]
- [0.0, 0.0]
- [0.0, 0.0]
- [0.0, 0.0]
- [0.0, 0.0]
- [0.0, 0.0]
sl_length_b: 0.3751358542967293
sl_length_r: 0.3720099458387134
